CodeEdit/InspectorSidebar/InspectorSidebarToolbar.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  InspectorSidebarToolbar.swift
+//  CodeEdit
+//
+//  Created by Austin Condiff on 3/21/22.
+//
+
+import SwiftUI
+
+struct InspectorSidebarToolbarTop: View {
+    @Binding var selection: Int
+
+    var body: some View {
+        HStack(spacing: 10) {
+            icon(systemImage: "doc", title: "File Inspector", id: 0)
```

Comment:
if we use that enum I was talking about here it would end up being like:
`icon(systemImage: "doc", title: "File Inspector", id: .fileInspector)` 